team_description_paper/introduction.tex

```diff
@@ -1,7 +1,13 @@
-Tech United Eindhoven\footnote{\url{http://www.techunited.nl}} is the RoboCup student team of Eindhoven University of Technology\footnote{\url{http://www.tue.nl}} that (since 2005) successfully competes in the robot soccer Middle Size League (MSL) and later (2011) also joined the ambitious @Home League. The Tech United @Home team has multiple vice World championship titles, two in the last three years, and is the vice European champion of the 2018 RoboCup German Open. %The robot soccer middle-size Tech United team has an even more impressive track record with four world championship titles.See the Tech United website for more results.
-Tech United Eindhoven consists of (former) PhD and MSc. students and staff members from different departments within the Eindhoven University of Technology.
-\\\\
-Many parts of our software are interacting with the world-model. Our world-model, \acrlong{ed}, is a database with 3D representations of objects. This is described in section \ref{sec:ed}. Other topics described in this paper are image recognition, pose detection, sound source localisation, Human-Robot interaction, Software sharing and community contributions.
-\\\\
-The previous years our focus has been on our own robots, AMIGO and SERGIO. This year we are shifting our focus to the Toyota HSR.
+Tech United Eindhoven\footnote{\url{http://www.techunited.nl}} is the RoboCup student team of Eindhoven University of Technology\footnote{\url{http://www.tue.nl}} (TU/e) that (since 2005) which joined the @Home League in 2011. 

```

Comment:
This sentence is somewhat off grammar-wise, can change to:
- "... (TU/e), which joined the @Home League in 2011."
- or the old version: "Eindhoven University of Technology\footnote{\url{http://www.tue.nl}} that (since 2005) successfully competes in the robot soccer Middle Size League (MSL) and later (2011) also joined the ambitious @Home League."

- or something like: "... (TU/e), that competes in the ambitious @Home League since 2011."

I like the last option, still short as well.